app/services/state_file/state_information_service.rb

```diff
@@ -202,7 +202,7 @@ def state_intake_class_names
         vita_link_es: "",
         voucher_form_name: "NJ Voucher Form",
         voucher_path: "",
-        w2_supported_box14_codes: ["UI_WF_SWF", "UI_HC_WD", "FLI"],
+        w2_supported_box14_codes: ["UI_WF_SWF", "FLI"],
```

Comment:
oh good catch, will do

Reply:
I actually decided to end up keeping this, because it all maps back to the DB. It could be renamed, but would require a db migration to rename the field there. (And there were about 70 uses of this throughout the app so I didn't want this diff to grow). And it is only important to not have underscores in the XML alone. If we still want to do that db field name change and related rename I'm not against it but a follow-up would be better in my opinion.